test/TouchGFX/gui/src/screen1_screen/Screen1Presenter.cpp

```diff
@@ -0,0 +1,59 @@
+#include <gui/screen1_screen/Screen1View.hpp>
```

Comment:
Preimenuj ovaj presener i view u nešto tipa mainMenuPresenter i mainMenuView, ovako je malo zbunjujuće.

Reply:
riješeno